cardano-cli/src/Cardano/CLI/Types/Errors/StakeCredentialError.hs

```diff
@@ -0,0 +1,19 @@
+{-# LANGUAGE LambdaCase #-}
+
+module Cardano.CLI.Types.Errors.StakeCredentialError
+  ( StakeCredentialError(..)
+  ) where
+
+import           Cardano.Api
+
+import           Cardano.CLI.Read
+
+data StakeCredentialError
+  = StakeCredentialScriptDecodeError (FileError ScriptDecodeError)
+  | StakeCredentialInputDecodeError (FileError InputDecodeError)
+  deriving Show
```

Comment:
Instead of using a command error, we use an error for stake credentials instead.